<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActuatorSettings</class>
 <widget class="QWidget" name="ActuatorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Actuator Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="listWidgetActuators">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="title">
        <string>Actuator Details</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="buddy">
           <cstring>lineEditName</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelProtocol">
          <property name="text">
           <string>Protocol:</string>
          </property>
          <property name="buddy">
           <cstring>comboBoxProtocol</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxProtocol">
          <item>
           <property name="text">
            <string>MÃ¤rklin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NMRA/DCC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selectrix</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>By Server</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBoxPort">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxAddress">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxPort</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelAddress">
          <property name="text">
           <string>Address:</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxAddress</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetActuators</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxProtocol</tabstop>
  <tabstop>spinBoxAddress</tabstop>
  <tabstop>spinBoxPort</tabstop>
  <tabstop>buttonNew</tabstop>
  <tabstop>buttonRemove</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonNew</sender>
   <signal>clicked()</signal>
   <receiver>ActuatorSettings</receiver>
   <slot>newActuator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemove</sender>
   <signal>clicked()</signal>
   <receiver>ActuatorSettings</receiver>
   <slot>deleteCurrent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newActuator()</slot>
  <slot>deleteCurrent()</slot>
  <slot>saveChanges()</slot>
 </slots>
</ui>
